app/migrations/Version20250805095503.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\PreUpAssertionMigration;
+use Mautic\PointBundle\Entity\Group;
+
+final class Version20250805095503 extends PreUpAssertionMigration
+{
+    protected const TABLE_NAME = Group::TABLE_NAME;
+
+    public function preUpAssertions(): void
+    {
+        $this->skipAssertion(
+            fn (Schema $schema) => $schema->getTable($this->getPrefixedTableName())->hasColumn('uuid'),
+            'Column uuid already exists'
+        );
+    }
+
+    public function up(Schema $schema): void
+    {
+        $table = $schema->getTable($this->getPrefixedTableName());
+        $table->addColumn('uuid', 'guid', ['nullable' => true]);
+        $this->addSql("UPDATE `{$this->getPrefixedTableName()}` SET `uuid` = UUID() WHERE `uuid` IS NULL");
```

Comment:
My bad, I just noticed an issue.

It should be `notnull => false` instead of `nullable => true`:
https://github.com/doctrine/dbal/blob/ce594cbc39a4866c544f1a970d285ff0548221ad/src/Schema/Column.php#L36-L37

However, even with that change, the schema still executes the statements in the wrong order. To avoid that, it’s better to use two explicit `addSql()` calls (tested and works):

```suggestion
        $this->addSql(sprintf(
            'ALTER TABLE `%s` ADD `uuid` CHAR(36) NULL DEFAULT NULL',
            $this->getPrefixedTableName()
        ));
        $this->addSql(sprintf(
            'UPDATE `%s` SET `uuid` = UUID() WHERE `uuid` IS NULL',
            $this->getPrefixedTableName()
        ));
```